RELEASE-NOTES.md

```diff
@@ -46,6 +46,7 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
   delimiter characters
 - Inlet: Added a method to `inlet::Inlet` that retrieves the set of unexpected names
   in the input file
+- Added support for registering material and species sets in `MFEMSidreDataCollection`
```

Comment:
Please expand this a bit, e.g.
```suggestion
- Added support for registering material and species sets in `MFEMSidreDataCollection`. These correspond to `matset` and `specset`s in the Mesh Blueprint. 
```
With the following links:
* https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh.html#material-sets
* https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh.html#species-sets